app/models/user.rb

```diff
@@ -1,17 +1,31 @@
 class User < ApplicationRecord
-  # Include default devise modules. Others available are:
-  # :confirmable, :lockable, :timeoutable and :omniauthable
+  enum status: [:enabled, :disabled]
+  enum role: [:user, :manager, :admin]
+
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :trackable, :validatable
+         
+  scope :all_users_except_admin, -> { all.where.not(role: :admin) }
```

Comment:
except_admin here would have been fine too...always think of the context when naming. 
'all.' is not required before where clause.

lets see these two....
User.all_users_except_admin
User.except_admin

User kind-off signifies that we are dealing with user entity, no need for 'all_users'

Reply:
Renamed and all removed from query.